apps/betterangels-backend/shelters/migrations/0009_remove_common_attachment_from_shelter.py

```diff
@@ -0,0 +1,58 @@
+from django.db import migrations
+
+
+def modify_shelter_group_permissions(apps, schema_editor):
+    Group = apps.get_model("auth", "Group")
+    Permission = apps.get_model("auth", "Permission")
+    ContentType = apps.get_model("contenttypes", "ContentType")
+
+    from common.permissions.enums import AddressPermissions
+
+    Address = apps.get_model("common", "Address")
+
+    # Data Entry Group
+    data_entry_permission_map = {
+        Address: [AddressPermissions],
+    }
+    group, created = Group.objects.get_or_create(name="Shelter Data Entry")
+
+    for model, perm_enums in data_entry_permission_map.items():
+        content_type = ContentType.objects.get_for_model(model)
+        for perm_enum in perm_enums:
+            for perm in perm_enum:
+                try:
+                    permission, created = Permission.objects.get_or_create(
```

Comment:
incredibly minor nit. also lines 35 and 42. but feel free to ignore
```suggestion
                    permission, _ = Permission.objects.get_or_create(
```